src/scss/components/_select.scss

```diff
@@ -273,12 +273,12 @@
       border-radius: 100%;
     }
   }
+}
 
-  .blip-select__option__content {
-    margin-left: 10px;
+.blip-select__option__content {
+  margin-left: 10px;
 
-    span {
-      display: block;
-    }
+  span {
+    display: block;
```

Comment:
If you're using `display: block` on a span, why not just use a `div` then?

Reply:
I've thought about it, but as long it's legacy code and it's used in the entire portal I've realized that it's wise don't change it for now